content/en/docs/reference/access-authn-authz/certificate-signing-requests.md

```diff
@@ -78,15 +83,18 @@ that case, the documentation for the signer should specify the meaning of
 additional certificates; for example, this might be the certificate plus
 intermediates to be presented during TLS handshakes.
 
-The PKCS#10 signing request format doesn't allow to specify a certificate
+The PKCS#10 signing request format does not allow to specify a certificate
 expiration or lifetime. The expiration or lifetime therefore has to be set
-through e.g. an annotation on the CSR object. While it's theoretically
```

Comment:
nit: PKCS#10 as a format does allow specifying this (via an extension); the problem is that there's not standard OID for recording the expiration date or lifetime. (I think we're saying that this separate mechanism makes more sense for Kubernetes than requiring users to learn how to use some custom CNCF OID for that, with all the ASN.1 fun that would entail - and I agree!